data/json/items/comestibles/raw_veggy.json

```diff
@@ -128,21 +128,21 @@
   {
     "type": "COMESTIBLE",
     "id": "carrot",
-    "name": { "str": "carrot" },
-    "weight": "68 g",
+    "name": { "str": "carrot", "str_pl": "carrots"},
```

Comment:
You don't need to do this if the plural is just +s, you can do undo the name change.

Reply:
Thanks, didn't know that. Fixed the names.